pp/integrations/plugin-packs/procedures/cloud-aws-fsx.md

```diff
@@ -5,190 +5,459 @@ title: Amazon FSx
+| --period            | Set period in seconds.                                                                                                                                                                                                                |
+| --timeframe         | Set timeframe in seconds.                                                                                                                                                                                                             |
+| --statistic         | Set cloudwatch statistics (Can be: 'minimum', 'maximum', 'average', 'sum').                                                                                                                                                           |
+| --zeroed            | Set metrics value to 0 if none. Useful when CloudWatch does not return value when not defined.                                                                                                                                        |
+| --timeout           | Set timeout in seconds (Default: 50).                                                                                                                                                                                                 |
+| --sudo              | Use 'sudo' to execute the command.                                                                                                                                                                                                    |
+| --command           | Command to get information (Default: 'aws'). Can be changed if you have output in a file.                                                                                                                                             |
+| --command-path      | Command path (Default: none).                                                                                                                                                                                                         |
+| --command-options   | Command options (Default: none). Only use for testing purpose, when you want to set ALL parameters of a command by yourself.                                                                                                          |
+| --proxyurl          | Proxy URL if any                                                                                                                                                                                                                      |
+| --skip-ssl-check    | Avoid certificate issuer verification. Useful when AWS resources are hosted by a third-party.  Note that it strips all stderr from the command result. Will be enhanced someday. Debug will only display CLI instead of evreything.   |
```

Comment:
```suggestion
| --skip-ssl-check    | Avoid certificate issuer verification. Useful when AWS resources are hosted by a third party.  Note that it strips all stderr from the command result. Will be enhanced someday. Debug will only display CLI instead of evreything.   |
```